aten/src/ATen/cuda/CUDAGeneratorImpl.h

```diff
@@ -87,9 +92,34 @@ namespace at {
  *
  */
 
+struct CUDAGeneratorState : public c10::intrusive_ptr_target {
+  uint64_t seed_;
+  uint64_t philox_offset_per_thread_;
+  uint32_t offset_intragraph_;
+  bool capturing_{};
+  std::unique_ptr<at::Tensor> seed_extragraph_;
+  std::unique_ptr<at::Tensor> offset_extragraph_;
```

Comment:
I am uncertain if employing `std::unique_ptr<at::Tensor>` is the optimal approach for this structure. Ideally, this member should simply be of type `at::Tensor`. However, directly using `at::Tensor` within the header file activates the `TORCH_ASSERT_NO_OPERATORS` error.